coverage-tests.js

```diff
@@ -1604,6 +1606,36 @@ test('check window fully with html scrollRootElement after scroll', {
   }
 })
 
+test('check coerce long image', {
```

Comment:
I think we need to invest more thoughts on the test name. Right now I would never understand what this test actually does. As for me, it should be called something like `check very long region by selector fully`, then I would easily understand what the point of this test.

Reply:
@kyryloonufriiev I disagree. Coerce means to force something to a limit. The name you suggested doesn't convey that. I would expect it to be just like any other long page, which'll render that test useless.

Reply:
Maybe let's do something mix, for example  `check coerce very long region by selector fully`. @itaibh  do you agree?

Reply:
We could but it would be redundant because there's no meaning to how we got that long or large image, only that we need to coerce it.

Reply:
@itaibh Ah, does this test check if we "crop" an image when it's bigger than some allowed size?

Reply:
@itaibh I find the word coerce confusing in this context. Can we be more verbose and say "force image to a limit" instead?

Reply:
Well, you know English better than me, so I'll take your word for it and agree.
Why do you find it confusing, may I ask?

Reply:
@tourdedave @kyryloonufriiev @YuriiChere Let me suggest the names `check limit to maximum image size for long page` and ``check limit to maximum image size for large page`.

Reply:
@tourdedave @kyryloonufriiev @itaibh I've implemented Itai's suggestion. It is in new PR https://github.com/applitools/sdk.coverage.tests/pull/39  . It was created instead of this one because there is an extra commit of Amit Zur in current PR. The new one is the same but with only my updates for BigPage tests. Please continue discussion in new PR and agree new tests' names or propose updates